data/campaigns/Winds_of_Fate/units/Dolphin.cfg

```diff
@@ -0,0 +1,91 @@
+#textdomain wesnoth-wof
+
+[unit_type]
+    id=Dolphin
+    name= _ "Dolphin"
+    race=aquatic
```

Comment:
Shouldn't this be `race=monster`?

Reply:
Dolphins don't really fit the concept of a "monster", though other animals in mainline are of the monster race…

Reply:
I agree with you, but since there is no `aquatic` race in core, it should be `monster`, or a new race has to be added.

Reply:
It is worth noting that a piglet is considered a monster while a gryphon is not. So there is a lot of leeway to class the dolphin as a monster or as a new race, at the moment.

Reply:
Moved the dolphin into the monster race for now.